ParseLiveQuery/build.gradle

```diff
@@ -8,11 +8,11 @@ version = '1.0.1-SNAPSHOT'
 
 buildscript {
     repositories {
-        mavenCentral()
+        jcenter()
```

Comment:
jcenter is a superset of maven central.

Android Studio by default has moved to jcenter

https://blog.bintray.com/2015/02/09/android-studio-migration-from-maven-central-to-jcenter/